docs/docs/guides/how-to-compose-your-own-testnet.md

```diff
@@ -57,16 +56,11 @@ Next, create a file called `network_params.json` in that folder with the followi
 ```
 The contents above will be used to define the specific parameters with which to start the network with.
 
-Now, create a `kurtosis.yml` file in the same folder with the following contents, replacing `$YOUR_GITHUB_USERNAME` with your actual Github username.
+Now, create a `kurtosis.yml` file in the same folder with the following:
 ```yml
-name: github.com/$YOUR_GITHUB_USERNAME/my-testnet
+name: github.com/my-testnet
```

Comment:
@leeederek why are the previous steps getting removed? The new stuff isn't correct; package names are `github.com/ORG/REPO` so this won't pass